router/transformer/transformer.go

```diff
@@ -200,22 +201,19 @@ func (trans *HandleT) ProxyRequest(ctx context.Context, responseData integration
 		var requestError error
 		//start
 		rdl_time := time.Now()
-		respData, respCode, requestError = trans.makeHTTPRequest(ctx, url, payload)
-		if requestError != nil {
-			stats.NewTaggedStat("transformer_proxy.request_latency", stats.TimerType, stats.Tags{"requestSuccess": "false"}).SendTiming(time.Since(rdl_time))
-			stats.NewTaggedStat("transformer_proxy.request_result", stats.CountType, stats.Tags{"requestSuccess": "false"}).Increment()
-		} else {
-			stats.NewTaggedStat("transformer_proxy.request_latency", stats.TimerType, stats.Tags{"requestSuccess": "true"}).SendTiming(time.Since(rdl_time))
-			stats.NewTaggedStat("transformer_proxy.request_result", stats.CountType, stats.Tags{"requestSuccess": "true"}).Increment()
-		}
+		respData, respCode, requestError = trans.makeHTTPRequest(ctx, url, payload, destName)
+		stats.NewTaggedStat("transformer_proxy.delivery_response", stats.CountType, stats.Tags{"destination": destName}).Increment()
+		reqSuccessStr := strconv.FormatBool(requestError != nil)
+		stats.NewTaggedStat("transformer_proxy.request_latency", stats.TimerType, stats.Tags{"requestSuccess": reqSuccessStr, "destination": destName}).SendTiming(time.Since(rdl_time))
+		stats.NewTaggedStat("transformer_proxy.request_result", stats.CountType, stats.Tags{"requestSuccess": reqSuccessStr, "destination": destName}).Increment()
 		//end
 		return requestError
 	}
 
 	backoffWithMaxRetry := backoff.WithMaxRetries(backoff.NewExponentialBackOff(), uint64(retryWithBackoffCount))
 	err = backoff.RetryNotify(operation, backoffWithMaxRetry, func(err error, t time.Duration) {
 		pkgLogger.Errorf("[Transformer Proxy] Request for proxy to URL:: %v, Error:: %+v retrying after:: %v,", url, err, t)
-		stats.NewStat("transformer_proxy.retry_metric", stats.CountType).Increment()
+		stats.NewTaggedStat("transformer_proxy.retry_metric", stats.CountType, stats.Tags{"destination": destName}).Increment()
```

Comment:
Rename it for clarity. 
```suggestion
		stats.NewTaggedStat("transformer_proxy.retry_metric", stats.CountType, stats.Tags{"destination": destName}).Increment()
```

Reply:
Yes